### PR TITLE
README.md for fe-angular

### PR DESCRIPTION
**Pull request by ultrasonicsoft**
README.md for fe-angular

### CODE REVIEW

File: boilerplates/fe-angular/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# fe-angular
+
+Angular quick starter project
```
Comment (michaelsauter) [inline boilerplates/fe-angular/README.md]: We use the `quickstarter` spelling throughout ...

File: boilerplates/fe-angular/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+ ## Purpose of this quickstarter
```
Comment (michaelsauter) [inline boilerplates/fe-angular/README.md]: If you remove the leading space, then Github diff will show some highlighting :)

File: boilerplates/fe-angular/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+│   ├── polyfills.ts
+│   ├── styles.scss
+│   ├── test.ts
+│   ├── tsconfig.app.json
+│   ├── tsconfig.spec.json
+│   └── tslint.json
+├── tsconfig.json
+└── tslint.json
+
+7 directories, 31 files
```
Comment (michaelsauter) [inline boilerplates/fe-angular/README.md]: How will this be kept up-to-date? Maybe we should condense it to highlight only the main folders/files?

File: boilerplates/fe-angular/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+ ## Frameworks used
+
+This project is generated by Angular CLI. (https://cli.angular.io/)
+ * Angular CLI version `6.0.7`
+ * Angular version `^6.0.3`
+ * Typescript version - `2.8.1`
```
Comment (michaelsauter) [inline boilerplates/fe-angular/README.md]: How will those version be kept up-to-date? I would probably remove them ...

File: boilerplates/fe-angular/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+│   └── tslint.json
+├── tsconfig.json
+└── tslint.json
+
+7 directories, 31 files
+```
+
+ ## Frameworks used
+
+This project is generated by Angular CLI. (https://cli.angular.io/)
```
Comment (michaelsauter) [inline boilerplates/fe-angular/README.md]: `by [Angular CLI](https://cli.angular.io/).` (that generates a clickable link)

File: boilerplates/fe-angular/README.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+The Jenkinsfile is provisioned with this quick starter to ease CI/CD process.
+In Jenkinsfile, there are various stages
+  * stageBuild - Builds the application by running `yarn install`, `yarn build` command and copies output folder `dist` into `docker/dist` folder.
+
+  * stageUnitTest - Runs unit test cases by executing command `yarn test`.
+  * stageLint - Runs `ng lint` profiler by running command `npm run lint`.
+  * stageScanForSonarqube - Runs Sonarqube on current project
+  * stageStartOpenshiftBuild - Start building application on OpenShift
+  * stageDeployToOpenshift - Deploy output to OpenShift
```
Comment (michaelsauter) [inline boilerplates/fe-angular/README.md]: I would only comment on `stageBuild`, `stageUnitTest` and `stageLint` - all other stages are standard shared lib

File: boilerplates/fe-angular/README.md (hunk 3 above)
Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: I think we can keep it consistent to generate this tree structure whenever we change angular CLI version. 
If all quickstarters project use `tree` output then we will have consistent file structure across ODS.

File: boilerplates/fe-angular/README.md (hunk 1 above)
Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: done!

File: boilerplates/fe-angular/README.md (hunk 4 above)
Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: Done!

File: boilerplates/fe-angular/README.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
+# fe-angular
+
+Angular quick starter project
+
+ ## Purpose of this quickstarter
+ 
+ This quick starter gives you a boilerplate code for new Angular project. This project contains basic setup for docker, jenkins, sonarqube and OpenShift.
```
[the remark this answers is not in the document]
Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: Done!

File: boilerplates/fe-angular/README.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
+# fe-angular
```
Comment (clemensutschig) [inline boilerplates/fe-angular/README.md]: should be Frontend Angular (fe-angular)

File: boilerplates/fe-angular/README.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+Angular quickstarter project
+
+## Purpose of this quickstarter
+ 
+ This quick starter gives you a new Angular project. This project contains basic setup for docker, jenkins, sonarqube and OpenShift. 
+
+## What files / architecture is generated?
+ ```
+.
+├── Jenkinsfile
```
Comment (clemensutschig) [inline boilerplates/fe-angular/README.md]: I think we should add a comment next to thow - e.g. JenkinsFile - this is the file that controls the build

File: boilerplates/fe-angular/README.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+│   ├── Dockerfile
+│   └── nginx.vh.default.conf.nginx
+├── e2e
+│   ├── protractor.conf.js
+│   ├── src
+│   │   ├── app.e2e-spec.ts
+│   │   └── app.po.ts
+│   └── tsconfig.e2e.json
+├── package.json
+├── sonar-project.properties
```
Comment (clemensutschig) [inline boilerplates/fe-angular/README.md]: same here # this is the file that controls the sonarqube scan plugin

Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: added

File: boilerplates/fe-angular/README.md (hunk 9 above)
Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: added

File: boilerplates/fe-angular/README.md (hunk 8 above)
Comment (ultrasonicsoft) [reply boilerplates/fe-angular/README.md]: changed

**Pull request merged**